components/style/values/generics/grid.rs

```diff
@@ -138,19 +138,25 @@ define_css_keyword_enum!{ TrackKeyword:
     "min-content" => MinContent
 }
 
-#[derive(Clone, Debug, PartialEq)]
-#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 /// A track breadth for explicit grid track sizing. It's generic solely to
 /// avoid re-implementing it for the computed type.
 ///
 /// https://drafts.csswg.org/css-grid/#typedef-track-breadth
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[derive(Clone, Debug, PartialEq, ToComputedValue)]
 pub enum TrackBreadth<L> {
     /// The generic type is almost always a non-negative `<length-percentage>`
     Breadth(L),
     /// A flex fraction specified in `fr` units.
-    Flex(CSSFloat),
+    Flex(
+        #[compute(clone)]
+        CSSFloat,
+    ),
     /// One of the track-sizing keywords (`auto`, `min-content`, `max-content`)
-    Keyword(TrackKeyword),
+    Keyword(
+        #[compute(clone)]
+        TrackKeyword,
```

Comment:
I think it'd be easier to just `impl ComputedValueAsSpecified for TrackKeyword` in this case, any reason that cannot be done?